cypress/integration/eventStatus.cy.js

```diff
@@ -103,4 +110,38 @@ describe('event status', () => {
 
         assertTooltipContainsEntries(['Scheduled'])
     })
+
+    it('can be filtered by status ACTIVE', () => {
+        setUpTable(event[DIMENSION_ID_LAST_UPDATED])
```

Comment:
This in combination with 
``` expectTableToMatchRows([`${getCurrentYearStr()}-05-03`]) ```
below is causing the test to fail when the row has been updated in the db, as the date is updated dynamically. I'll switch this to use one of the static periods instead (event date etc)